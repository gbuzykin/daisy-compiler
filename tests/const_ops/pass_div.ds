#if (2678i32 / 99871i32) != 0i32
#error invalid result
#endif
#if (2678u32 / 99871i32) != 0u32
#error invalid result
#endif
#if (47i8 / 99871i32) != 0i32
#error invalid result
#endif
#if (47u8 / 99871i32) != 0i32
#error invalid result
#endif
#if (2678u32 / 99871u32) != 0u32
#error invalid result
#endif
#if (47i8 / 99871u32) != 0u32
#error invalid result
#endif
#if (47u8 / 99871u32) != 0u32
#error invalid result
#endif
#if (-2678i32 / -99871i32) != 0i32
#error invalid result
#endif
#if (4294964618u32 / -99871i32) != 1u32
#error invalid result
#endif
#if (-47i8 / -99871i32) != 0i32
#error invalid result
#endif
#if (209u8 / -99871i32) != 0i32
#error invalid result
#endif
#if (4294964618u32 / 4294867425u32) != 1u32
#error invalid result
#endif
#if (-47i8 / 4294867425u32) != 1u32
#error invalid result
#endif
#if (209u8 / 4294867425u32) != 0u32
#error invalid result
#endif
#if (-2678i32 / 99871i32) != 0i32
#error invalid result
#endif
#if (4294964618u32 / 99871i32) != 43005u32
#error invalid result
#endif
#if (-47i8 / 99871i32) != 0i32
#error invalid result
#endif
#if (209u8 / 99871i32) != 0i32
#error invalid result
#endif
#if (4294964618u32 / 99871u32) != 43005u32
#error invalid result
#endif
#if (-47i8 / 99871u32) != 43005u32
#error invalid result
#endif
#if (209u8 / 99871u32) != 0u32
#error invalid result
#endif
#if (2678i32 / -99871i32) != 0i32
#error invalid result
#endif
#if (47i8 / -99871i32) != 0i32
#error invalid result
#endif
#if (47u8 / -99871i32) != 0i32
#error invalid result
#endif
#if (47i8 / 4294867425u32) != 0u32
#error invalid result
#endif
#if (47u8 / 4294867425u32) != 0u32
#error invalid result
#endif
