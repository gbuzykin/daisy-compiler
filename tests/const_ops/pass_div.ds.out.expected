./const_ops/pass_div.ds: info: warnings 0, errors 0
